go.mod

```diff
@@ -38,14 +38,14 @@ require (
 	golang.org/x/crypto v0.6.0
 	golang.org/x/mod v0.9.0
 	golang.org/x/text v0.8.0
-	helm.sh/helm/v3 v3.10.3
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/client-go v0.25.4
-	k8s.io/component-base v0.25.2
+	helm.sh/helm/v3 v3.11.2
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.0
+	k8s.io/client-go v0.26.0
+	k8s.io/component-base v0.26.0
```

Comment:
Limited by CAPI v1.3.5 using k8s.io/api v0.25.0.
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3279 on hold for the [same reason](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3279#discussion_r1134054083)